server/src/server.ts

```diff
@@ -110,7 +110,7 @@ class Server {
           hoverProvider: true,
           documentRangeFormattingProvider: true,
           documentHighlightProvider: true,
-          foldingRangeProvider: true,
+          //foldingRangeProvider: true,
```

Comment:
just set it to false instead of introducing a comment.

Reply:
done! Was not sure if it was maybe intended to be available, and folding also works anyway in monaco (regardless of support in the language server), but sure, when it's not supported anyway, false is the better/correct option